ssa/utils.go

```diff
@@ -448,6 +448,7 @@ func fixHorizontalPodAutoscaler(object *unstructured.Unstructured) error {
 	if object.GetKind() == "HorizontalPodAutoscaler" {
 		switch object.GetAPIVersion() {
 		case "autoscaling/v2beta2":
+		case "autoscaling/v2":
 			var d hpav2beta2.HorizontalPodAutoscaler
```

Comment:
Do we want to continue to use the `hpav2beta2` here instead of moving to `hpav2`?

Reply:
Good catch! I think we might can to do `hpav2beta2` for `autoscaling/v2beta2` and then `hpav2` for `autoscaling/v2`. I am not sure if we can do that without some duplication in the two cases

Reply:
That sounds OK to me.